package_build_list_host_windows.json

```diff
@@ -7,7 +7,7 @@
         "assimp-5.2.5-rev1-windows":  "Scripts/extras/pull_and_build_from_git.py ../../package-system/assimp --platform-name Windows --package-root ../../package-system --clean",
         "astc-encoder-3.2-rev2-windows": "Scripts/extras/pull_and_build_from_git.py ../../package-system/astc-encoder --platform-name Windows --package-root ../../package-system --clean",
         "azslc-1.8.15-rev2-windows": "Scripts/extras/pull_and_build_from_git.py ../../package-system/azslc --platform-name Windows --package-root ../../package-system/azslc/temp --clean",
-        "AWSGameLiftServerSDK-3.4.2-rev1-windows": "package-system/AWSGameLiftServerSDK/build_package_image.py --platform-name windows",
+        "AWSGameLiftServerSDK-5.0.0-rev1-windows": "package-system/AWSGameLiftServerSDK/build_package_image.py --platform-name windows",
```

Comment:
will updating this entry (vs. adding a new one) affect users' ability to build the existing AWSGameLift gem, which currently relies on server SDK 3.4.2?

Reply:
No, this will not affect building the AWSGameLift gem as that is using the GameLift Server SDK that is already uploaded to cloudfront.

The `package_build_list_host_<platform>.json` files here is used by the the 3p-package-scripts [build_package.py](https://github.com/o3de/3p-package-scripts#script-build_packagepy) script to build the actual 3rd Party from source.
That built library is then pushed to the production by sig-build once it is ready for use.
But the existing libraries remain in production.

Reply:
Correct, Derric beat me to it :)